.github/workflows/rust-clippy.yml

```diff
@@ -29,6 +29,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      
+      - uses: Swatinem/rust-cache@v1
```

Comment:
hmm what does this do again

Reply:
it caches all of the packages and possible the toolchain so that it doesn't have to download it each time

Reply:
ooooh nice